apps/namadillo/src/App/Ibc/IbcTransfer.tsx

```diff
@@ -35,6 +35,29 @@ export const IbcTransfer: React.FC = () => {
   const performIbcTransfer = useAtomValue(ibcTransferAtom);
   const defaultAccounts = useAtomValue(allDefaultAccountsAtom);
 
+  const transactionFee = useMemo(() => {
+    if (typeof registry !== "undefined") {
+      // TODO: can we get a better type for registry to avoid optional chaining?
+      // TODO: some chains support multiple fee tokens - what should we do?
+      const feeToken = registry.chain?.fees?.fee_tokens?.[0];
+
+      if (typeof feeToken !== "undefined") {
+        const asset = registry.assets.assets.find(
+          (asset) => asset.base === feeToken.denom
+        );
+
+        if (typeof asset !== "undefined") {
+          return {
+            amount: BigNumber(1), // TODO: remove hardcoding
```

Comment:
We probably just need a follow-up issue to handle fees when submitting to Keplr right?

Reply:
I think so! I've adapted #1190 to track this.